user_data/strategies/test_strategy.py

```diff
@@ -239,8 +304,142 @@ def populate_sell_trend(self, dataframe: DataFrame) -> DataFrame:
         dataframe.loc[
             (
                 (dataframe['adx'] > 70) &
-                (dataframe['tema'] > dataframe['bb_middleband']) &
                 (dataframe['tema'] < dataframe['tema'].shift(1))
             ),
             'sell'] = 1
         return dataframe
+
+    def hyperopt_space(self) -> List[Dict]:
```

Comment:
While i agree of having hyperopt arguments as part of the strategy file - is it then also read as part of the hyperopt.py file? 
i think this depends on having #571 merged first - or am i missing something?

Reply:
I believe so, but, however, that PR does look pretty nice.

Really really really need that PR to be able to not need to manually move over the test_strategy into default_strategy (I think at first glance).

Either way works. This PR can be kept as it is or merged or closed, no worries :) I just wanted to document some of the learning process it took to get hyperopt -s all running correctly.

However, I noticed my hyperopt does not output any better strategies past about 100 epochs.

Any clue? I could run 5K epochs and get no better strategy optimizations.